packages/node-resolve/src/cjs-wrapper.js

```diff
@@ -0,0 +1,6 @@
+import nodeResolve, { DEFAULT_OPTIONS } from './index';
+
+const wrapper = nodeResolve;
+wrapper.DEFAULT_OPTIONS = DEFAULT_OPTIONS;
```

Comment:
```js
wrapper.DEFAULT_OPTIONS = deepFreeze(defaults);
```

This should export a deeply frozen copy of defaults. We don't want the risk of a consumer modifying the defaults.